src/lib/operators/table_scan.cpp

```diff
@@ -146,8 +146,6 @@ std::shared_ptr<const Table> TableScan::_on_execute() {
             }
           }
 
-          if (pos_list_in->references_single_chunk()) filtered_pos_list->guarantee_single_chunk();
```

Comment:
Intended for this PR?

Reply:
Should have left a review comment for that. Yes, it is repeated from a few lines above.